engine/src/flutter/lib/web_ui/analysis_options.yaml

```diff
@@ -14,5 +14,4 @@ linter:
     no_default_cases: false
     prefer_relative_imports: false
     public_member_api_docs: false
-    unawaited_futures: true
```

Comment:
Why this change?

Reply:
There were zero instances of this lint being hit!

Reply:
Because it was enabled by the line you are removing, preventing anyone from adding violations of the rule. The fact that the rule was working as intended isn't a reason to remove it.

Reply:
Gah! Yeah. I like putting ignores up top and then a list of lints at the bottom to make this more clear. Will fix.